src/component/content.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from "react";
+
+const firstName = ['Yoda','Willy','fool','dilly','billy','Sam', ''];
+
+const lastName =['Sanchez','silly','meow','son','Roger'];
+
+const renderFirst= firstName[Math.floor(Math.random()* firstName.length)];
+    
+const rendeLast= lastName[Math.floor(Math.random()* lastName.length)];
+   
+const renderboth = renderFirst + " " + rendeLast;
+
+
+
+
+
+class Content extends Component{
+    constructor(props){
+        super(props);
+        this.state={};
```

Comment:
state should contain the currently generated name
```suggestion
        this.state={
          randomName: '',
        };
```